ModTek/ModTek.csproj

```diff
@@ -21,7 +21,7 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>none</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>..\</OutputPath>
```

Comment:
I actually have the git directly checked out as Mods/ModTek, so the dll would be put where I expect it to load after having built it

Reply:
Please move this into `ModTek.csproj.user` for yourself as opposed to changing everyone's workflow.